src/algebra/squarefree.lean

```diff
@@ -422,6 +430,58 @@ lemma squarefree_iff_prime_sq_not_dvd (n : ℕ) :
 squarefree_iff_irreducible_sq_not_dvd_of_exists_irreducible
   ⟨2, (irreducible_iff_nat_prime _).2 prime_two⟩
 
+lemma squarefree_mul {m n : ℕ} (hmn : m.coprime n) :
+squarefree m ∧ squarefree n ↔ squarefree (m * n) :=
+begin
+  split,
```

Comment:
Mathlib style is to use `{}` after split to make it clear which tactics are operating on which direction; so in this case, the proof would proceed as:
```lean
split,
{ rintros ⟨hm, hn⟩,
  -- solve the forward direction here
  sorry },
{ intros hmn_sq,
  -- solve the backwards direction here
  sorry },
```

Reply:
also, you can use `refine ⟨_, λ a, _⟩` to do the `split` and `intros` at the same time (sadly this doesn't work with `rintro`)

Reply:
Thanks for that! Neat trick. Though bhavik's suggestion obviates all of this :-)